schema.graphql

```diff
@@ -432,15 +446,86 @@ type Match {
   notes: String
 }
 
+"""
+Docs group: Question
+"""
+type Packet {
+  """
+  A human-readable identifier. Often primarily numeric (e.g., `Packet 1`), but could also be identified by authors (`Chicago` or `Yale + Maryland`), as for packet-submission tournaments.
+  """
+  name: String!
+  
+  """
+  The packet number, if that notion is meaningful. May also be contained in the `name`, but need not be, e.g. if both `number` and `authors` are specified, the `name` will typically only contain one of them (whichever is more important).
+  """
+  number: Int
+  
+  """
+  The authors of the packet. May also be contained in the `name`, but need not be, e.g. if both `number` and `authors` are specified, the `name` will typically only contain one of them (whichever is more important).
+  """
+  authors: [String]
+
+  """
+  How the packet is expected to be used. An `extra` packet typically consists only of tossups, and is for use in overtime and/or as replacements, as necessary. Note that a packet might be used differently in case of unusual situations.
+  """
+  role: PacketRole
+
+  """
+  Any further details on the packet.
+  """
+  notes: String
+
+  """
+  The contents of the packet, in order.
+  """
+  questions: [Question]
+}
+
+enum PacketRole {
+  regular
+  finals
+  extra
+  overtime
+  replacement
```

Comment:
Is `replacement` significantly different from `extra`? What about `backup` and `tiebreaker`? Should some of these values have short definitions/blurbs?